spec/httpserver/handlers/ok_spec.clj

```diff
@@ -0,0 +1,11 @@
+(ns httpserver.ok-spec
+  (:require [speclj.core :refer :all]
+            [httpserver.handlers.ok :as ok]))
+
+(import '(com.td.HttpServer HttpResponse))
+
+(describe "OkHandler"
+  (it "returns an HttpRequest with code 200"
+    (let [ok-handler (ok/new-ok-handler)
+           response (.generateResponse ok-handler)]
```

Comment:
FYI, if you highlight a block of code in vim visual mode and press `=` it should fix all indentation issues